src/widget/wkey.cpp

```diff
@@ -56,8 +56,6 @@ void WKey::setCents() {
     setValue(m_dOldValue);
 }
 
-void WKey::preferencesUpdated(double dValue) {
-    if (dValue > 0) {
-        setValue(m_dOldValue);
-    }
+void WKey::keyNotationChanged(double dValue) {
+    setValue(m_dOldValue);
```

Comment:
warning: unused parameter ‘dValue’ [-Wunused-parameter]

Reply:
Ups, ... yes. this value should be used. Fixed.